src/widgetastic/widget.py

```diff
@@ -373,7 +373,9 @@ def cls_widget_names(cls):
         for key in dir(cls):
             value = getattr(cls, key)
             if isinstance(value, Widgetable):
-                result.append((key, value))
+                # check the values of a VersionPick object are widgetable themselves
+                if all([isinstance(item, Widgetable) for item in value.child_items]):
```

Comment:
When value.child_items returns an empty list (default implementation in Widgetable.child_items) this statement evaluates to True. Don't ask me why `all([])` is True though.